java/code/src/com/redhat/rhn/manager/content/MgrSyncUtils.java

```diff
@@ -251,28 +255,37 @@ public static URI urlToFSPath(String urlString, String name) {
         // Case 2
         if (OFFICIAL_UPDATE_HOSTS.contains(host)) {
             mirrorPath = new File(dataPath.getAbsolutePath(), path);
+            LOG.info("SCC mirrorpath: {}", mirrorPath);
         }
         else if (name != null) {
             // Case 3
             // everything after the first space are suffixes added to make things unique
-            String[] parts  = URLDecoder.decode(name, StandardCharsets.UTF_8).split("[\\s//]");
+            String[] parts  = URLDecoder.decode(name, StandardCharsets.UTF_8).split("[\\s/]");
             if (!(parts[0].isBlank() || parts[0].equals(".."))) {
                 File oldMirrorPath = Paths.get(dataPath.getAbsolutePath(), "repo", "RPMMD", parts[0]).toFile();
+                LOG.info("SMT mirrorpath for '{}': {}", name, oldMirrorPath);
```

Comment:
Is this log statement at the right spot or was it supposed to be in the if statement?

Reply:
This is the right place. We want to show the "options" where we look on the mirror. Users might not know where to put the stuff. So it does not exist. Putting it in the if would only show it when they found the right location.